tests/network/services.go

```diff
@@ -78,6 +78,26 @@ var _ = SIGDescribe("Services", func() {
 		}
 	}
 
+	assertConnectionToExposedServiceSucceeds := func(serviceName, namespace string, servicePort int) {
```

Comment:
You asked to keep the details on `runTCPClientExpectingHelloWorldFromServer` but now you drop it.

Reply:
No, I just moved them elsewhere. 

Is there an action point here ?

Reply:
Not sure what you mean by "moved them".
I commented that `runTCPClientExpectingHelloWorldFromServer` is to informative in its name, you mentioned that it is something you want to have. But now you wrap it, and reduce the information (i.e. abstract it).

And IMO, the name here is a bit complex, giving more "how" than "what".
Maybe this will be a bit more clear: "assertConnectivityToService".

Reply:
Done.